### PR TITLE
Integrated the Channels library

### PR DESCRIPTION
**Pull request by davidluca**

### CODE REVIEW

**Comment by lucifurtun** [comment]
@ralucamiclea, @davidluca, what's the progress on this PR ? :)

**Comment by davidluca** [comment]
we are currently working on consumers.

**Comment by lucifurtun** [comment]
I would suggest to not create a different app only to setup channels. All the setup code can live in `project` package. Afterwards, we can move the vehicle related code to the future vehicle app.

File: liveServer/consumers.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        await self.channel_layer.send(
+            self.vehicle_group_name,
+            {
+                'type': 'vehicle_id',
+                'message': message
+            }
+        )
+
+    async def chat_message(self, event):
+        message = event['message']
```
Comment (lucifurtun) [inline liveServer/consumers.py]: We don't need this chat functionality. All we need is the option to push data from server to client (web browser).

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Django == 2.0.4
 djangorestframework == 3.8.2
-djangorestframework-jwt == 1.11.0
\ No newline at end of file
+djangorestframework-jwt == 1.11.0
+channels == 2.1.1
+
```
Comment (lucifurtun) [inline requirements.txt]: Do not forget to add `channels_redis` as requirement.

File: liveServer/consumers.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+from channels.generic.websocket import AsyncWebsocketConsumer
+import json
+
+class VehicleConsumer(AsyncWebsocketConsumer):
```
Comment (lucifurtun) [inline liveServer/consumers.py]: @davidluca, does this config work ?

**Pull request merged**